src/util.js

```diff
@@ -32,10 +32,15 @@ export const flattenDeep = (list /* : any[] */) /* : any[] */ =>
     list.reduce((memo, x) => memo.concat(Array.isArray(x) ? flattenDeep(x) : x), []);
 
 const UPPERCASE_RE = /([A-Z])/g;
-const MS_RE = /^ms-/;
-
-const kebabify = (string /* : string */) /* : string */ => string.replace(UPPERCASE_RE, '-$1').toLowerCase();
-export const kebabifyStyleName = (string /* : string */) /* : string */ => kebabify(string).replace(MS_RE, '-ms-');
+const UPPERCASE_RE_TO_KEBAB = (match /* : string */)  /* : string */ => `-${match.toLowerCase()}`;
+
+export const kebabifyStyleName = (string /* : string */) /* : string */ => {
+  const result = string.replace(UPPERCASE_RE, UPPERCASE_RE_TO_KEBAB);
+  if (result.slice(0, 3) === 'ms-') {
```

Comment:
I made a maybe-more-useful microbenchmark that actually tests strings that start with `ms-` and it also agrees (much more strongly actually) so this looks great! :) https://jsperf.com/slice-vs-bracket-string-check-ms